WeightProducer/python/MCSampleValues.py

```diff
@@ -84,422 +84,624 @@ class MCSampleValuesHelper():
         "TTJets_DiLept" : {
             "CrossSection" : XSValues(XS_13TeV=88.34),
         },
-        # MET binned: GenXSecAnalyzer scaled by ttbar inclusive k-factor (NNLO vs LO), 831.76/511.3 = 1.627
+        # MET binned
         "TTJets_SingleLeptFromT_genMET-150" : {
-            "CrossSection" : XSValues(XS_2016=9.683904),
-            "kFactor" : kFactorValues(kFactor_2016=1.627),
+            "CrossSection" : XSValues(XS_13TeV=5.952, XSSource_13TeV="XSDB (only available for 2016)"),
+            "kFactor" : kFactorValues(kFactor_2016=1.627, kFactorSource_2016="XSDB NNLO/LO=831.76/511.3",
```

Comment:
this indentation is terrible, please fix it to correspond with everything else i.e.:
```
            "kFactor" : kFactorValues(
                kFactor_2016=1.627, kFactorSource_2016="XSDB NNLO/LO=831.76/511.3",
                kFactor_2017=1.677, kFactorSource_2017="XSDB NNLO/LO=831.76/496.1",
                kFactor_2018=1.677, kFactorSource_2018="Same as 2017",
            ),
```